package.json

```diff
@@ -27,41 +29,33 @@
     "svg.js": "^2.6.5"
   },
   "scripts": {
-    "prepublish": "mastarm prepublish lib:build",
-    "cover": "npm test -- --coverage",
-    "lint": "mastarm lint lib",
+    "build": "tsdx build -i lib/transitive.ts --target node",
+    "cover": "tsdx test --coverage --collectCoverageFrom=lib/**/*",
+    "deploy-storybook": "storybook-to-aws-s3 --bucket-path=transitive.js --s3-sync-options=--acl=public-read",
+    "jest": "tsdx test",
+    "lint": "tsdx lint __tests__ lib stories .storybook --quiet",
+    "prepublish": "yarn build",
     "semantic-release": "semantic-release",
     "start": "start-storybook -p 5555",
-    "deploy-storybook": "storybook-to-aws-s3 --bucket-path=transitive.js --s3-sync-options=--acl=public-read"
+    "test": "yarn run lint && yarn jest"
   },
   "devDependencies": {
-    "@babel/core": "^7.12.3",
-    "@babel/preset-env": "^7.12.1",
     "@opentripplanner/base-map": "^1.0.4",
     "@opentripplanner/core-utils": "^3.0.2",
-    "@storybook/addon-essentials": "^6.1.0-beta.6",
-    "@storybook/addon-info": "5.3.21",
-    "@storybook/addon-links": "^6.1.0-beta.6",
-    "@storybook/preset-create-react-app": "^3.1.5",
-    "@storybook/react": "^6.1.0-beta.6",
-    "@storybook/storybook-deployer": "^2.8.7",
-    "@webpack-cli/init": "^1.0.3",
-    "babel-loader": "^8.1.0",
-    "core-js": "2",
-    "css-loader": "^5.0.1",
-    "file-loader": "^6.2.0",
+    "@storybook/addon-essentials": "^6.3.7",
+    "@storybook/addon-links": "^6.3.7",
+    "@storybook/react": "^6.3.7",
+    "@storybook/storybook-deployer": "^2.8.10",
+    "@types/component-emitter": "^1.2.10",
+    "@types/d3": "^3.5.8",
+    "eslint-plugin-prettier": "^3.4.0",
     "leaflet": "^1.7.1",
     "lodash.isequal": "^4.5.0",
-    "mastarm": "^5.0.1",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
     "react-leaflet": "^2.6.1",
     "semantic-release": "^17.2.3",
-    "style-loader": "^2.0.0",
     "styled-components": "^5.2.1",
-    "terser-webpack-plugin": "^5.0.3",
-    "webpack": "^5.4.0",
-    "webpack-cli": "^4.2.0"
-  },
-  "standard": {
-    "parser": "babel-eslint"
+    "tsdx": "https://github.com/ibi-group/tsdx#jest-transform-change"
```

Comment:
We should probably look into getting this pushed to master right?

Reply:
that would be ideal.